mapstruct-protobuf3/spi-impl/src/main/java/org/mapstruct/example/protobuf/ProtobufAccessorNamingStrategy.java

```diff
@@ -0,0 +1,34 @@
+package org.mapstruct.example.protobuf;
+
+import org.mapstruct.ap.spi.DefaultAccessorNamingStrategy;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.type.TypeMirror;
+
+/**
+ * @author Thomas Kratz
+ */
+public class ProtobufAccessorNamingStrategy extends DefaultAccessorNamingStrategy {
+
+    public static final String PROTOBUF_GENERATED_MESSAGE_V3 = "com.google.protobuf.GeneratedMessageV3";
+    public static final String LIST_SUFFIX = "List";
+
+    @Override
+    public String getElementName(ExecutableElement adderMethod) {
+
+        String methodName = getPropertyName(adderMethod);
```

Comment:
I didn't notice this before., but you are calling `getPropertyName(adderMethod)` instead of `super.getElementName(adderMethod)`. It currently works, because in `getPropertyName()` we always strip the first 3 characters. However, I think that it is more correct if you just call `super.getElementName(adderMethod)`